sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -259,6 +260,10 @@ public String getAvailableMemory() {
         public String getSystemCpuLoad() {
             return systemCpuLoad;
         }
+
+        public Integer getAvailableProcessors() {
```

Comment:
why not primitive? can this ever be null?

Reply:
ACK